src/main/webapp/app/assessment-shared/assessment-header/assessment-header.component.ts

```diff
@@ -33,4 +35,11 @@ export class AssessmentHeaderComponent {
     @Output() cancel = new EventEmitter<void>();
     @Output() resolveConflict = new EventEmitter<void>();
     @Output() nextSubmission = new EventEmitter<void>();
+
+    constructor(private accountService: AccountService, private route: ActivatedRoute) {}
+
+    ngOnInit() {
+        this.isAtLeastInstructor = this.accountService.hasAnyAuthorityDirect(['ROLE_ADMIN', 'ROLE_INSTRUCTOR']);
```

Comment:
This does not make sure that the instructor is an instructor of this course.

Reply:
Also it should probably be an input as the component will only be used in the `assessment-layout.component`. Then we could both remove the accountService and the route (see comment above), which would make the component dumb and easy to test.

Reply:
Done.